content/tutorial/tutorial.md

```diff
@@ -976,30 +983,30 @@ Finally, we need to move the `handleClick` method from the Board component to th
   }
 ```
 
->Note
+>ध्यान दें
 >
->Unlike the array `push()` method you might be more familiar with, the `concat()` method doesn't mutate the original array, so we prefer it.
+>Array के विपरीत `push()` मेथड से आप अधिक परिचित हो सकते हैं, `concat()` मेथड मूल array को म्यूटेट नहीं करती है, इसलिए हम इसे पसंद करते हैं।
 
-At this point, the Board component only needs the `renderSquare` and `render` methods. The game's state and the `handleClick` method should be in the Game component.
+इस बिंदु पर, Board कौम्पोनॅन्ट को केवल `renderSquare` और `render` मेथड की आवश्यकता है। Game कौम्पोनॅन्ट के state में गेम का state और `handleClick` मेथड होनी चाहिए।
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/EmmOqJ?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/EmmOqJ?editors=0010)**
 
-### Showing the Past Moves {#showing-the-past-moves}
+### पिछली चाल दिखाना {#showing-the-past-moves}
 
-Since we are recording the tic-tac-toe game's history, we can now display it to the player as a list of past moves.
+चूंकि हम टिक-टैक-टो गेम की history को रिकॉर्ड कर रहे हैं, इसलिए हम इसे पिछले चालों की सूची के रूप में खिलाड़ी को प्रदर्शित कर सकते हैं।
 
-We learned earlier that React elements are first-class JavaScript objects; we can pass them around in our applications. To render multiple items in React, we can use an array of React elements.
+हमने पहले सीखा था कि React एलिमेंट्स प्रथम श्रेणी के जावास्क्रिप्ट ऑब्जेक्ट हैं; हम उन्हें अपने अनुप्रयोगों में पास कर सकते हैं। React में कई वस्तुओं को प्रस्तुत करने के लिए, हम React एलिमेंट्स की एक array का उपयोग कर सकते हैं।
```

Comment:
अनुप्रयोगों ?

Reply:
React में कई वस्तुओं को प्रस्तुत करने के लिए, हम React एलिमेंट्स की एक array का उपयोग कर सकते हैं। -> revisit

Reply:
sounds fine to me.
just changed `प्रस्तुत` => `रेंडर`

Reply:
ऑब्जेक्ट हैं; aur हम उन्हें अपने ऍप्लिकेशन्स में पास कर ...